glfs.c

```diff
@@ -495,13 +499,14 @@ static int tcmu_glfs_open(struct tcmu_device *dev)
 
 	gfsp->fs = tcmu_create_glfs_object(dev, config, &gfsp->hosts);
 	if (!gfsp->fs) {
-		tcmu_dev_err(dev, "tcmu_create_glfs_object failed\n");
+		tcmu_dev_err(dev, "tcmu_create_glfs_object(config=%s) failed\n", config);
 		goto fail;
 	}
 
 	gfsp->gfd = glfs_open(gfsp->fs, gfsp->hosts->path, ALLOWED_BSOFLAGS);
 	if (!gfsp->gfd) {
-		tcmu_dev_err(dev, "glfs_open failed: %m\n");
+		tcmu_dev_err(dev, "glfs_open(vol=%s, file=%s) failed: %m\n",
+		             gfsp->hosts->volname, gfsp->hosts->path);
```

Comment:
One FYI and one review comment.

The runner code normally follows the kernel coding style which is like above where the next line lines up with the "(". In your patch you seem to be changing that to where it lines up with the first quote, but here you used the runner style.

I do not really care about coding style in your own code as long as its not too far from the style used in the core code, but you should use one style.

Reply:
> I do not really care about coding style in your own code as long as its not too far from the style used in the core code, but you should use one style.

tcmu_dev_log(dev, "%s %s",
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;x, y);
I read it like x & y are passed to fill "%s %s"   or rather x & y are input to fill "%s %s"

But that may be my own perception, I'm making changes to align with runner style :-)